### PR TITLE
Migrate tracker to angular router.

### PR DESCRIPTION
**Pull request by tan9**

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/admin/_admin.module.ts [hunk 1]
```diff
@@ -33,7 +36,7 @@ import {
     <%_ } _%>
     <%_ if (websocket === 'spring-websocket') { _%>
     <%=jhiPrefixCapitalized%>TrackerComponent,
-    TrackerResolve,
+    trackerRoute,
```
Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/app/admin/_admin.module.ts]: `trackerRoute` is not needed to import here.

File: generators/client/templates/angular/src/main/webapp/app/admin/_admin.module.ts [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { RouterModule } from '@angular/router';
 import { ParseLinks } from 'ng-jhipster';
+<%_ if (websocket === 'spring-websocket') { _%>
+import { JhiTrackerService } from './../shared/tracker/tracker.service';
```
Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/app/admin/_admin.module.ts]: change this to `<%=jhiPrefixCapitalized%>TrackerService`

File: generators/client/templates/angular/src/main/webapp/app/admin/_admin.module.ts [hunk 3]
```diff
@@ -91,8 +94,8 @@ import {
         <%_ } _%>
         LogsService,
         <%_ if (websocket === 'spring-websocket') { _%>
-        TrackerResolve,
-        <%_ } _%>        
+        JhiTrackerService,
```
Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/app/admin/_admin.module.ts]: same here as above

File: generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { Injectable, Inject } from '@angular/core';
 import { Observable, Observer, Subscription } from 'rxjs/Rx';
-import { UIRouter } from 'ui-router-ng2';
+import { Router, NavigationEnd } from '@angular/router';
```
Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts]: can you just move this above? such that we will group `@angular` imports together

**Comment by tan9** [comment]
@sendilkumarn , commit revised according to your comments. Thank you.

**Pull request merged**